src/main/java/seedu/taskitty/logic/commands/SaveCommand.java

```diff
@@ -0,0 +1,63 @@
+package seedu.taskitty.logic.commands;
+
+import static seedu.taskitty.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import java.io.IOException;
+
+import seedu.taskitty.commons.util.ConfigUtil;
+import seedu.taskitty.commons.util.StringUtil;
+import seedu.taskitty.storage.StorageManager;
+import seedu.taskitty.commons.core.Config;
+import seedu.taskitty.commons.exceptions.IllegalValueException;
+
+//@@author A0135793W
+/**
+ * Saves TasKitty data to a folder specified by the user
+ * @author JiaWern
+ *
+ */
+public class SaveCommand extends Command{
+    
+    public static final String COMMAND_WORD = "save";
+
+    public static final String MESSAGE_PARAMETER = COMMAND_WORD
+            + " filepath";
+    public static final String MESSAGE_USAGE = "This command saves data in TasKitty to a location of your choice, Meow!\n"
```

Comment:
MESSAGE_USAGE is too long for us to fit. Try to cut it down to 1 or 2 lines. Just basic info. The rest of the info, the user will have to check the UserGuide